### PR TITLE
Frontend Notification Modal is Created

### PR DESCRIPTION
**Pull request by UkcaGreen**
Notification modal is created. 
In order to implement this following components are used:
- Modal component from Ant Design
- Notification component that is created from scratch

Screenshot:
resim

### CODE REVIEW

**Comment by UkcaGreen** [comment]
I am planning to convert navigation links into buttons, what you think?

Example screenshot:
resim

**Comment by aliramazanmert** [comment]
Again, after connecting to the backend, we can merge this pr.

File: frontend/src/components/MainHeader/index.jsx
Comment (cemreefe) [inline frontend/src/components/MainHeader/index.jsx:29]: We should connect to the backend before merging, since the necessary endpoints are implemented since this push. I am requesting a review.

**Review by cemreefe — CHANGES_REQUESTED** [review]
Everything seems to be correctly implemented. However this code is not connected to the backend and makes use of a dummy data. The request can be approved after the adequate backend connection.

**Comment by aliramazanmert** [comment]
We are expecting this new beautiful header buttons and notification modal(with backend connection and accepting/rejecting requests) to be ready asap.

**Review by cemreefe — APPROVED** [review]
I have reviewed the pr, there are no conflicts and the code seems fine. I am merging the branch.

**Pull request merged**
